app/models/user.rb

```diff
@@ -24,7 +24,9 @@
 class User < ApplicationRecord
   extend Enumerize
 
-  enumerize :role, in: %i[simple admin], default: :simple, scope: :having_role
+  enumerize :role, in: %i[simple admin], default: :simple,
+                   scope: :having_role,
+                   predicates: { prefix: true }
```

Comment:
```suggestion
  enumerize :role, in: %i[simple admin],
                   default: :simple,
                   scope: :having_role,
                   predicates: { prefix: true }
```
можна так, або взагалі в один рядок